### PR TITLE
No alert after clicking (un)deprecate

### PR DESCRIPTION
**Pull request by MilanPospisil**
Issue: AAH-1578

https://issues.redhat.com/browse/AAH-1578

Adding deprecation/undeprecation alert after deprecate/undeprecate button clicked.

### CODE REVIEW

File: src/containers/namespace-detail/namespace-detail.tsx [hunk 1]
```diff
@@ -377,7 +377,23 @@ export class NamespaceDetail extends React.Component<IProps, IState> {
           !collection.deprecated,
           this.context.selectedRepo,
         )
-          .then(() => this.loadCollections())
+          .then(() => {
+            let title;
+            !collection.deprecated
+              ? (title = t`The collection ${collection.name} has been deprecated.`)
```
Comment (ShaiahWren) [inline src/containers/namespace-detail/namespace-detail.tsx:383]:
```suggestion
              ? (title = t`Collection "${collection.name}" has been successfully deprecated.`)
```

**Review by ShaiahWren — COMMENTED** [review]
I don't think this is your doing in the pr, but the deprecate/undeprecate functionality is not working properly. The label only updates after the second click, resulting in two alerts of the same type. 

As for the alert wording, I've suggested a few minor changes to better match alerts in the rest of the the UI

Nice work:)

File: src/containers/namespace-detail/namespace-detail.tsx [hunk 2]
```diff
@@ -377,7 +377,23 @@ export class NamespaceDetail extends React.Component<IProps, IState> {
           !collection.deprecated,
           this.context.selectedRepo,
         )
-          .then(() => this.loadCollections())
+          .then(() => {
+            let title;
```
Comment (himdel) [inline src/containers/namespace-detail/namespace-detail.tsx:381]: also, what's up with the `let title; expression ? (title = foo) : (title = bar)` construct?

```suggestion
            const title = !collection.deprecated ? t`Collection ${collection.name} has been deprecated.` : t`Collection "${collection.name}" has been successfully undeprecated.`;
```

File: src/containers/namespace-detail/namespace-detail.tsx [hunk 3]
```diff
@@ -377,7 +377,23 @@ export class NamespaceDetail extends React.Component<IProps, IState> {
...
+            !collection.deprecated
+              ? (title = t`The collection ${collection.name} has been deprecated.`)
+              : (title = t`The collection ${collection.name} has been undeprecated.`);
+            this.setState({
+              alerts: [
+                ...this.state.alerts,
+                {
+                  id: 'deprecation/undeprecation',
```
Comment (himdel) [inline src/containers/namespace-detail/namespace-detail.tsx:389]: unused id?

File: src/containers/namespace-detail/namespace-detail.tsx [hunk 4]
```diff
@@ -377,7 +377,23 @@ export class NamespaceDetail extends React.Component<IProps, IState> {
...
+                  title: title,
+                  variant: 'success',
+                },
+              ],
+            });
+            this.loadCollections();
```
Comment (himdel) [inline src/containers/namespace-detail/namespace-detail.tsx:395]: missing return

**Comment by brumik** [comment]
This is nearly the same as #2067

File: src/containers/namespace-detail/namespace-detail.tsx (hunk 3 above)
Comment (MilanPospisil) [reply src/containers/namespace-detail/namespace-detail.tsx]: Im not sure if id is needed.

**Review by ZitaNemeckova — APPROVED** [review]
All previous comments has been addressed. Works in UI. LGTM 👍

**Pull request merged**